src/sentry/api/endpoints/organization_stats.py

```diff
@@ -51,7 +50,7 @@ def get(self, request, organization):
             raise ValueError("Invalid group: %s" % group)
 
         if "id" in request.GET:
-            id_filter_set = frozenset(map(int, request.GET.getlist("id")))
+            id_filter_set = frozenset(list(map(int, request.GET.getlist("id"))))
```

Comment:
Same here, can we do a mass transform here similar to set(list(...))

Reply:
This is done in an optimization pass that I linked in the description. I guess we'd prefer it to be all rolled up in 1 PR though. https://github.com/getsentry/sentry/pull/28080#issuecomment-908578834